src/pool/srv_iv.c

```diff
@@ -691,11 +740,9 @@ ds_pool_iv_prop_update(struct ds_pool *pool, daos_prop_t *prop)
 	pool_iv_prop_l2g(prop, &iv_entry->piv_prop);
 
 	rc = pool_iv_update(pool->sp_iv_ns, IV_POOL_PROP, iv_entry, size,
-			    CRT_IV_SHORTCUT_NONE, CRT_IV_SYNC_EAGER, false);
+			    CRT_IV_SHORTCUT_NONE, CRT_IV_SYNC_LAZY, true);
```

Comment:
some comments for the memory usage/free would be helpful, either here or in pool_iv_update or ds_iv_update()

Reply:
ok